### DISCUSSION TITLE
CLI usage possibilities were reduced?

### DISCUSSION THREAD

**Discussion by hoijui** [git-bug/git-bug#1236] [Q&A] [answered]
I just installed current master as of today, and am testing:

1. The docu mentions `git bug ls`, but is seems not to exist anymore. Is there an other way to list issues?
2. The docu mentions `git bug bridge configure`, but is seems not to exist anymore. -> I want to add a second bridge to a second github remote on a local repo. I opened .git/config, and copy&pasted the entry, editing the owner field accordingly; that seems to work somewhat, though when I push to this bridge, it pushed 0 issues (and there are no issues in the github UI), even though I have 54 open issues locally.
3. what is the difference between `git bug push` and `git bug bridge push` (and same for `pull`)? I know it, but purely from CLI help messages, I could not deduce it.
4. Is it possible to pull form one github remote/bridge and push to an other?
5. It is `git bug help` but `git bug bridge --help`

**Comment by MichaelMure** [comment]
This is a dup of https://github.com/MichaelMure/git-bug/issues/1209 (or the other way around). CLI has been refreshed and the documentation hasn't followed yet.

> Is it possible to pull form one github remote/bridge and push to an other?

No, git-bug prevent this (see https://github.com/MichaelMure/git-bug/issues/1223#issuecomment-2268697889) as it's almost always a bad idea. Possibly some special case could be carved out, but it wasn't the focus at the time.

**Comment by sudoforge** [comment, marked as answer]
Converting this to a Q&A discussion. The issue that was brought up -- outdated documentation -- will be tracked in #1209. To the remaining questions...

> The docu mentions git bug ls, but is seems not to exist anymore. Is there an other way to list issues?

If you've built from HEAD, `ls` was moved under a `bug` subcommand, so (_somewhat awkwardly_), you need to type `git-bug bug`. `git bug ls` is still appropriate if you're using `v0.8.0`.

---

> what is the difference between `git bug push` and `git bug bridge push` (and same for `pull`)? I know it, but purely from CLI help messages, I could not deduce it.

`git-bug-push` and `git-bug-pull` interact with `refs/bugs` namespace on the remote -- pushing and pulling like you to a branch under the `refs/heads` namespace.

`git-bug-bridge-push` and `git-bug-bridge-pull` interact with the bridge's API, and could more aptly be described as "import to git-bug" and "export from git-bug" operations. I agree that this experience can be improved. One thought I have is to make bridge operations transparent (#1223), but we could also provide better clarification via documentation and helptext.

---

> It is `git bug help` but `git bug bridge --help`

Thanks for bringing this inconsistency up.
